internal/main.go

```diff
@@ -86,6 +94,18 @@ func main() {
 	}
 
 	if !engine.Run(flags.stage.String()) {
+		if flags.oem.String() == "azure" {
+			err := azure.ReportProvisioningFailed("something went wrong")
```

Comment:
I meant to change that to "provisioning failed, please check boot logs for Ignition errors". Is there a better way to phrase that in the context of azure @crawford?

I'm also not sure where this text ends up going, so I should figure that out.